### PR TITLE
Suggestions and update on spec

### PR DESCRIPTION
**Pull request by amityadav0**
- comments

### CODE REVIEW

File: spec/README.md [hunk 1]
```diff
@@ -45,9 +45,11 @@ Guarantee of Desired Exchange: Users can trust that the contract ensures a fair
...
 1.  A maker cancels a previously created order. Expired orders can also be cancelled.
-2.  An Order can only be cancel once. 
-3.  Tokens should be refunded when order is cancel.
-4.  An order can be also cancel if there are unfinished bidding orders. 
+2.  An Order can only be cancelled once. 
+3.  Tokens should be refunded when order is cancelled.
+4.  An order can be also cancelled even if there are unfinished bidding orders. 
+
+Question: What does this mean ? I think maker should not be able to accept as they already cancelled order
 5.  Maker can accept the bid order if the order has canceled.
```
Comment (liangping) [inline spec/README.md]: can *not*
please help to improve this sentence.

File: spec/README.md [hunk 2]
```diff
@@ -108,14 +110,15 @@ interface CancelSwapMsg {
 
  - Anyone can bid for any open orders. if the order has specified desired recipient, then only he can bid for this old.
  - bid price should great than 0 and less than order price
+    - makers can set price limit for bids for their order. For example: minimum bid amount: 100
```
Comment (liangping) [inline spec/README.md]: Is bid amount quantity or price?

Comment (amityadav0) [reply spec/README.md]: price, let me add this

Comment (amityadav0) [reply spec/README.md]: because we don't support partial bid as of now

**Pull request merged**